app/templates/partials/jobs/status.html

```diff
@@ -20,4 +20,9 @@
       {% endif %}
     {% endif %}
   </p>
+  {% if job.status == 'sending limits exceeded'%}
+    <p class="govuk-error-message" >
+    You have exceeded your daily sending limits. You can upload your CSV tomorrow or <a class="govuk-link" href="https://www.notifications.service.gov.uk/support">contact the GOV.UK Notify team</a>
```

Comment:
Minor: I think we should avoid encouraging users to contact us about this, as we only want to be increasing the limit in special cases (where they can't wait until tomorrow). Would be good to get a second opinion from @karlchillmaid.

Reply:
We should also explain that the job will never be processed: it's ambiguous because just below the notifications are shown as "sending".

Reply:
My suggestion, adapting https://github.com/alphagov/notifications-admin/blob/cfe022bc7f673df28799e75422fff211460a194a/app/templates/partials/check/too-many-messages.html#L1-L23
```suggestion
    Notify cannot send these messages because you have reached your daily limit. You can only send {{ current_service.message_limit|format_thousands }} messages per day. Upload this spreadsheet again tomorrow or <a class="govuk-link" href="https://www.notifications.service.gov.uk/support">contact the GOV.UK Notify team</a> to raise the limit.
```

Reply:
@quis Would you like me to make the message for exceeding the daily limits the same? This is the message from the upload page, I had changed the message from the /jobs/{id} page. 
Current upload page error message: 
![too-many-recipients](https://user-images.githubusercontent.com/4282990/125400674-17e8a480-e3aa-11eb-95d2-c7d42a056016.png)

New error message on jobs page: 
![process_job_exceeded](https://user-images.githubusercontent.com/4282990/125400721-259e2a00-e3aa-11eb-82c4-1cc6af8bbf8d.png)

Reply:
I think that looks OK. Could be slightly improved by splitting into two paragraphs and adding some spacing, eg
```html
<p class="govuk-error-message">
    Notify cannot send these messages because you have reached your daily limit. You can only send {{ current_service.message_limit|format_thousands }} messages per day.
</p>
<p class="govuk-error-message govuk-!-margin-bottom-6">
    Upload this spreadsheet again tomorrow or <a class="govuk-link" href="https://www.notifications.service.gov.uk/support">contact the GOV.UK Notify team</a> to raise the limit.
</p>
```